yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -217,17 +219,15 @@ export async function computeVerificationKey(
     const logFunction = (message: string) => {
       log(`computeVerificationKey(${circuitName}) BB out - ${message}`);
     };
-    const args = ['-o', outputPath, '-b', bytecodePath, '-v', recursive ? '--recursive' : ''];
-    let result = await executeBB(pathToBB, `write_vk_${flavor}`, args, logFunction);
+    const args = ['-o', outputPath, '-b', bytecodePath, '-v'];
+    if (recursive) {
+      args.push('--init_kzg_accumulator');
+    }
+    const result = await executeBB(pathToBB, `write_vk`, args, logFunction);
```

Comment:
Is `--scheme` arg not needed for `write_vk` here?
earlier it used to be one of `write_vk_mega_honk`, `write_vk_ultra_honk` or `write_vk_ultra_keccak_honk`

Reply:
Oh looks like this `computeVerificationKey` method is not used anywhere

Reply:
Thanks for noticing--removed the function.